src/canvas.py

```diff
@@ -274,10 +275,18 @@ def on_draw_event(self, _widget, ctx):
             self._renderer.dpi = self.figure.dpi
             self.figure.draw(self._renderer)
 
+    @property
+    def visible_axes(self):
+        return self._visible_axes
+
+    @visible_axes.setter
+    def visible_axes(self, visible_axes):
+        self._visible_axes = visible_axes
+
```

Comment:
This is redundant. I'd make this either a GProperty or refer to the next comment.

Reply:
Ueaj, a GProperty would be better here.